x/precisebank/keeper/keeper.go

```diff
@@ -3,21 +3,43 @@ package keeper
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+
+	"github.com/kava-labs/kava/x/precisebank/types"
 )
 
-// TODO: Enforce that Keeper implements the expected keeper interfaces
-// var _ types.BankKeeper = Keeper{}
+// Enforce that Keeper implements the expected keeper interfaces
+var _ evmtypes.BankKeeper = Keeper{}
```

Comment:
Yeah I originally was using precisebank expected keeper interface, but realized they're different purposes even though they have mostly the same methods:
- evmtypes.BankKeeper - x/evm requirements so we need to define those methods in x/precisebank, some of these could be removed though since they're not used in x/evm
- types.BankKeeper - x/precisebank requirements that are slightly different like needing x/bank `BlockedAddr` later on, this method doesn't need to be implemented on x/precisebank keeper, only consumed

Reply:
makes sense! seems fine to keep the dependency as it expresses exactly what is needed.
could also see adding a `EvmBankKeeper` interface but i dont think the duplication is worth it just to remove the dependency.

if we run into DevX problems with precisebank's reliance on x/evm as a dependecy we can reevaluate.